packages/esm-service-queues-app/src/queue-entry-table-components/queue-priority.scss

```diff
@@ -9,3 +9,7 @@
 .bold {
   font-weight: bold;
 }
+
+.orange {
+  background-color: #ffc9a3;
```

Comment:
So, in general, we want to prefer Carbon's color tokens wherever possible rather than hard-coded hex values. In this case, maybe `layout.$orange-20`, which is not identical, but pretty close.

Reply:
fixed